e2e-tests/trade-finance-manager/cypress/integration/journeys/facility/facilities.spec.js

```diff
@@ -1,26 +1,40 @@
 import relative from '../../relativeURL';
 import MOCK_DEAL_AIN from '../../../fixtures/deal-AIN';
+import MOCK_DEAL_MIA from '../../../fixtures/deal-MIA';
 import MOCK_USERS from '../../../fixtures/users';
 import { MOCK_MAKER_TFM } from '../../../fixtures/users-portal';
 import partials from '../../partials';
 import page from '../../pages';
 
 context('Facility page', () => {
-  let dealId;
-  const dealFacilities = [];
+  let dealOne;
+  let dealTwo;
+  const dealOneFacilities = [];
+  const dealTwoFacilities = [];
 
   before(() => {
-    cy.deleteTfmDeals(dealId);
-
+    cy.deleteTfmDeals(dealOne?._id);
+    cy.deleteTfmDeals(dealTwo?._id);
```

Comment:
seems there's no need to pass anything here, just a single `cy.deleteTfmDeals();` should work?